react/src/base/inputs/SprkTextarea/SprkTextarea.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import uniqueId from 'lodash/uniqueId';
+
+const SprkTextarea = ({
+  additionalClasses,
+  analyticsString,
+  forwardedRef,
+  idString,
+  isDisabled,
+  id,
+  value,
+  formatter,
+  onChange,
```

Comment:
We have been passing this in through `...rest`. We should talk about whether we want it as a prop and add it to the other places or remove it here.